### PR TITLE
Minor lint tweaks

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,22 +28,22 @@ module.exports = function(src, options = {}) {
   const dependencies = [];
 
   if (typeof src === 'undefined') {
-    throw new Error('src not given');
```
Comment (mrjoelkemp) [inline index.js:32]: Would this be considered a breaking change?

Comment (XhmikosR) [reply index.js:32]: Not sure, it depends. It's still an error, its type is more explicit is all.

Comment (XhmikosR) [reply index.js:32]: We can revert it if you prefer, I just have my linter set up this way (xo).

Comment (mrjoelkemp) [reply index.js:32]: I think it's safer to revert this particular change. Someone trying to sniff for Error won't catch the TypeError unfortunately.

Comment (XhmikosR) [reply index.js]: Done, just squash and merge when ready :)

**Review by mrjoelkemp — APPROVED** [review]

**Pull request merged**